client-src/elements/utils.js

```diff
@@ -298,8 +298,8 @@ export function formatFeatureChanges(fieldValues, featureId) {
   // Multiple stages can be mutated, so this object is a stage of stages.
   const stages = {};
   for (const {name, touched, value, stageId, implicitValue} of fieldValues) {
-    // Only submit changes for touched fields.
-    if (!touched) {
+    // Only submit changes for touched fields or accuracy verification updates.
+    if (!touched && (name !== 'accurate_as_of' || value !== true)) {
```

Comment:
I'm not sure about the `|| value !== true part`.  Does that make it send every false checkbox every time?  Will that add noise to the activity log?

Reply:
Good catch - updated to fix that case